private/buf/bufformat/testdata/proto2/option/v1/option.golden.proto

```diff
@@ -93,16 +93,16 @@ enum Type {
       // Trailing comment on message literal.
 
       // Leading comment on foo.
-      foo: 1, // Trailing comment on foo.
+      foo: 1 // Trailing comment on foo.
       // Leading comment on bar.
       bar: 2 // Trailing comment on bar.
       // Leading comment on recursive.
-      recursive: <
+      recursive: {
         // Leading comment on foo.
-        foo: 1,
+        foo: 1
```

Comment:
Personally I prefer always including the trailing commas even if they aren't necessary (perhaps a side effect of use of Go structs, JSON objects, etc.). It could be argued though that the least amount of unnecessary whitespace is desirable.